.gitignore

```diff
@@ -9,6 +9,7 @@ __pycache__/
 # Distribution / packaging
 .Python
 env/
+e-env/
```

Comment:
Is it necessary?

Reply:
It should have no influence on our code, right? It is my local environment name.